printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md

```diff
@@ -0,0 +1,92 @@
+
+
+
+This mod is for the V2, it involves printing new AB drive frames, new XY Joints, new Front Tensioners, and new Z idlers
+
+The pins have replaced m5 screws, at any location where the screw acts like a shaft, holding bearings, pulleys or idlers. This allows for smoother rotation and stops the possibility of motion parts being caught in threads. This work was done by everyone, thanks to ABS plastic and voron printers for the fast iteration, @doomweasel? they fall out yet?
+
+To complete this mod the following files will need to be reprinted 
+
+**A and B drive units**
+
+ - a_drive_frame_lower_pinned.stl 
+ - a_drive_frame_upper_pinned.stl 
+ - b_drive_frame_lower_pinned.stl 
+ - b_drive_frame_upper_pinned.stl
+
+**Front Idlers**
+- [a]_a_tensioner_pinned.stl
+- [a]_b_tensioner_pinned.stl
+
+**XY Joints**
+- xy_joint_left_upper_pinned.stl
+- xy_joint_left_lower_pinned.stl
+- xy_joint_right_upper_pinned.stl
+- xy_joint_right_lower_pinned.stl
+
+**Z Idlers**
```

Comment:
This directory also includes `z_tensioner_bracker_{a,b}_x2.stl`, are those 2 STLs supposed to be printed too?   Or are they the same as stock?  If same as stock, then remove them from the mod.

Reply:
these have also been removed